Handle connection_init and connection_terminate

fdev opened the pull request:
When using Apollo Client (specifically `subscriptions-transport-ws`) the [websocket client expects](https://github.com/apollographql/subscriptions-transport-ws/blob/567fca89cb4578371877faee09e1630dcddff544/src/client.ts#L614) either a `connection_ack` or `connection_error` response from the server after sending the initial `connection_init` message.

Currently, `graphene-subscriptions` just ignores these message types which causes the client to never emit `connected` events and may cause a `maxConnectTime` trigger to be performed.

fdev committed:
Handle connection_init and connection_terminate

jaydenwindle commented:
Thanks a ton for the PR @fdev! I'd love to merge this in.

Would you mind adding some unit tests as well?

fdev committed:
Fix message, add tests

fdev commented:
Thanks for the quick response @jaydenwindle. I have added tests and squashed a bug in the process.

By the way, I'm getting warnings about the `id` variable overriding a built-in Python function.

qwaqwa93TW commented:
@jaydenwindle Why is this PR not being merged? I am suffering from that timeout problem

wnaldj1589TW commented:
Is anyone watching this issue?
